project.clj

```diff
@@ -4,7 +4,7 @@
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
   :dependencies [[org.clojure/clojure "1.8.0"]
-                 [org.clojure/core.rrb-vector "0.0.11"]]
+                 [org.clojure/core.rrb-vector "0.0.12"]]
```

Comment:
It doesn't look like this release has been cut yet. Feel free to let me know when it has been.

Reply:
it is already released 2 hours ago. search in maven central still shows only 0.0.11 version.
but if you run lein deps then 0.0.12 version will be downloaded.

Reply:
Tag for 0.0.12 : https://github.com/clojure/core.rrb-vector/releases/tag/core.rrb-vector-0.0.12